pkg/apis/eventing/v1/broker_defaults.go

```diff
@@ -36,7 +36,7 @@ func (b *Broker) SetDefaults(ctx context.Context) {
 
 func (bs *BrokerSpec) SetDefaults(ctx context.Context) {
 	cfg := config.FromContextOrDefaults(ctx)
-	c, err := cfg.Defaults.GetBrokerConfig(apis.ParentMeta(ctx).Namespace)
+	c, err := cfg.Defaults.GetBrokerConfig(apis.ParentMeta(ctx).Namespace, "")
```

Comment:
shouldn't we use the broker spec for this string?

Reply:
After I think more a bit, I think we should leave it blank here.

The function `SetDefaults` is used to set the default configuration when creating the new broker. From the code [here](https://github.com/knative/eventing/blob/d989ca732fad3701fef677f80594504512a39ee5/pkg/reconciler/testing/v1/broker.go#L36-L41) we can tell that no brokerClass is passed in. So according to the default rules, it will use the default brokerClass in the namespace config.